lib/broadside/target.rb

```diff
@@ -16,27 +16,16 @@ class Target
       :predeploy_commands,
       :scale,
       :service_config,
-      :tag,
```

Comment:
@matl33t actually this is an important question - i think in 2.0 i broke the ability to configure the `tag` per target - @camertron noticed that - and here i am fully removing that feature.  if we want that feature back, now would be the time.